src/test/java/reactor/netty/tcp/BlockingConnectionTest.java

```diff
@@ -91,14 +101,15 @@ public void simpleServerFromAsyncServer() throws InterruptedException {
 				         .wiretap(true)
 				         .bindNow();
 
-		System.out.println(simpleServer.address().getHostString());
-		System.out.println(simpleServer.address().getPort());
+		InetSocketAddress address = (InetSocketAddress) simpleServer.address();
+		System.out.println(address.getHostString());
```

Comment:
(just a note to remove the `System.out`)

Reply:
removed